tools/autograd/derivatives.yaml

```diff
@@ -345,10 +345,10 @@
   self: zeros_like(grad)
 
 - name: norm(Tensor self, Scalar p=2)
-  self: norm_backward(grad, self, p)
+  self: norm_backward(grad, self, p, result)
 
 - name: norm(Tensor self, Scalar p, int64_t dim, bool keepdim=False)
-  self: norm_backward(grad, self, p, dim, keepdim)
+  self: norm_backward(grad, self, p, destination, dim, keepdim)
```

Comment:
What's a destination?

Reply:
The names of returns values are taken from `Declarations.cwrap`. It's not visible to Python code.

I'd like to make them consistent: right now they're a mix of "result", "destination", and "output".  I'll probably rename them all to output unless there's a more meaningful name (like indices)